src/styles/untiles/_mixins.scss

```diff
@@ -0,0 +1,28 @@
+@mixin shadow {
+  height: 60px;
+  background: $white-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+@mixin stars-active {
+  @for $st from 1 through 4 {
+    &:nth-child(#{$st}) {
```

Comment:
check if your code is working

Reply:
you have a working example in the checklist
<img width="948" alt="image" src="https://user-images.githubusercontent.com/102411130/236767495-517cf202-6c71-4a98-bac2-67eff31ba66e.png">